shape.h

```diff
@@ -122,6 +122,12 @@ RUBY_SYMBOL_EXPORT_BEGIN
 RUBY_EXTERN rb_shape_tree_t rb_shape_tree;
 RUBY_SYMBOL_EXPORT_END
 
+static inline shape_id_t
+rb_shape_get_shape_count(void)
```

Comment:
nitpick:
```suggestion
rb_shapes_count(void)
```

Much shorter, avoid the repetition etc.

Reply:
Good call